### DISCUSSION TITLE
MAC OS

### DISCUSSION THREAD

**Discussion by chickenduden** [yermak/AudioBookConverter#582] [MacOS]
Doesn't work on High Sierra and Big Sur.
The program opens, allows you to download the files, but does not let you convert them. When you click on "start" nothing happens.

**Comment by chickenduden** [comment]
I have the steam version

**Comment by jessearmstrong** [comment]
Doesn't work on M1 Mac Studio either.

**Comment by yermak** [comment]
As a heads-up, I'm working on pure native version for Mac to overcome all those problems (I faced many others), as I tired to fight with Mac specific issues in current implementation - I spent all might time on mac specific issues.
Mac version will be launched very soon, it will be very basic at the begining, but hopefully all those issues will be not relevant.

**Comment by red-ochre** [comment]
Hi Yermak, Thankyou for a great product.
Could we please have an update on progress of releasing a macOS version?
Thanks

**Comment by katemyers** [comment]
Commenting as well, checking in on the promised mac version!

**Comment by yermak** [comment]
Duplicates #580

**Comment by yermak** [comment]
Beta for Mac OS X is available
https://github.com/yermak/AudioBookConverter/issues/580#issuecomment-2268738667
